src/Adyen/Client.php

```diff
@@ -10,7 +10,7 @@
 
 class Client
 {
-    const LIB_VERSION = "15.0.0-beta";
+    const LIB_VERSION = "15.1.0";
```

Comment:
Beta exit shall be done manually. 
```suggestion
    const LIB_VERSION = "15.0.0";
```